packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -7,6 +7,17 @@
       class="dropdown-title"
       type="button"
       :aria-label="dropdownAriaLabel"
+      @click="handleDropdown"
+    >
+      <span class="title">{{ item.text }}</span>
+      <span
+        class="arrow down"
+      />
+    </button>
+    <button
```

Comment:
The reason for adding a `mobile-dropdown-title` is that the mobile view (sidebar at the left) does not need to handle the tab event, right?